src/Nav/Nav.test.tsx

```diff
@@ -41,12 +41,13 @@ describe('Nav component', () => {
 	// })
 
 	describe('click to open menu', () => {
-		const expectedLinkRoutes = ['/', '/config', '/icons', '/search-users']
+		const expectedLinkRoutes = ['/', '/config', '/icons', '/search-users', '/display-users']
```

Comment:
Nice test update! 🤘🏻

Reply:
Wait, were you just pointing this out or do you want something changed?

Reply:
Just point it out, =]